spec/que/locker_spec.rb

```diff
@@ -185,6 +185,16 @@ def assert_que_locker_insertion(
     locker.stop!
   end
 
+  it "should create and delete pid file, if passed" do
+    pidfile = "./spec/temp/pidfile_#{Time.new.to_i}.pid"
+    locker_settings[:pidfile] = pidfile
+    locker
+    assert_equal true, File.exist?(pidfile)
```

Comment:
Style suggestion: you can instead call `assert` and `refute` for these two boolean assertions